src/platform/datapath_raw_xdp_linux.c

```diff
@@ -357,7 +357,7 @@ AttachXdpProgram(struct xdp_program *Prog, XDP_INTERFACE *Interface, struct xsk_
         unsigned int xdp_flag;
     } AttachTypePairs[]  = {
         // { XDP_MODE_HW, XDP_FLAGS_HW_MODE },
-        { XDP_MODE_NATIVE, XDP_FLAGS_DRV_MODE },
+        // { XDP_MODE_NATIVE, XDP_FLAGS_DRV_MODE },
```

Comment:
Why?

Reply:
DRV mode is not fully functional on Azure VM, but attachable to NIC without error. Normally there should be error if kenrel/hw doesn't support some features, then we can fallback to lower grade mode.
I'm asking accelerated networking team how to programmatically identify the functionality

Reply:
Ok.